personal/privacyidea/class_mfaAccount.inc

```diff
@@ -512,8 +512,8 @@ class mfaAccount extends plugin
 
         $this->smarty->assign("allTokenTypes", array_keys($this->mfaTokens));
 
-        $mfaRequiredByUser = $this->isCurrentUserUsertab() ? ($this->read_only ? "r" : "rwcdm") : $this->getacl("mfaRequired");
-        $this->smarty->assign("mfaRequiredACLorCurrentUser", $mfaRequiredByUser);
+        $mfaRequiredByUser = $this->isCurrentUserUsertab() ? ($this->read_only ? "r" : "rwcdm") : $this->getacl("mfaRequiredByUser");
+        $this->smarty->assign("mfaRequiredByUserACLorCurrentUser", $mfaRequiredByUser);
```

Comment:
The CurrentUser bit should be coverable by an ACL, as well. In GOsa² you can mark UI elements / attributes as editable (or whatever perm) by the owner of the object. So we very don't need the currentUser check here.

Reply:
The special casing for the current user has been removed.